src/machine/CoffeeMachine.java

```diff
@@ -0,0 +1,111 @@
+package machine;
+
+import java.util.Scanner;
+
+class CoffeeMachine {
+
+    private int amountOfWater;
+    private int amountOfMilk;
+    private int amountOfCoffeeBeans;
+    private int amountOfDisposableCups;
+    private int money;
+    private CoffeeMachineState currentState;
```

Comment:
minor: we can add an empty line before the `currentState ` to visually separate it from the other fields that represent the amount of remaining supplies.

Also we can think about extracting the supplies to a separate class, but it's not necessary at this stage